### PR TITLE
Various updates to installer.

### PR DESCRIPTION
**Pull request by computer-whisperer**
Added commands for downloading and installing opkg files, updated architecture and repo link for 2016.

### CODE REVIEW

**Comment by computer-whisperer** [comment]
If this is merged I'll pr it to pyfrc.

**Comment by virtuald** [comment]
This is great... but it doesn't appear to resolve dependencies?

**Comment by computer-whisperer** [comment]
Not for opkgs at the moment. I could look into doing that.

**Comment by virtuald** [comment]
I think it should be pretty straightforward to do now that you've added arbitrary opkg installation. The dependency data is in the Packages file, which we already download.

**Comment by virtuald** [comment]
Do you think you'll have time this week to do the dependency resolving?

**Comment by computer-whisperer** [comment]
I can try to put some time into it tomorrow. It shouldn't be that hard.

**Comment by computer-whisperer** [comment]
Finished, it can now resolve, download, and install dependencies -- including from the base NI repo.

**Comment by virtuald** [comment]
Awesome, I'll try it out tonight. The only nit I have is that I think you broke python 2.7 compatibility with the addition of the urllib import, but that should be easy enough to fix. 

Only reason I care about python 2.7 compatibility is I would like to position this to be used for general opkg installation, and it makes the barrier for entry a bit lower.

File: installer/installer.py [hunk 1]
```diff
@@ -546,14 +564,18 @@ class RobotpyInstaller(object):
     opkg_cache = abspath(join(dirname(__file__), 'opkg_cache'))
     
     # opkg feed
-    opkg_feed = 'http://www.tortall.net/~robotpy/feeds/2014/'
-    opkg_arch = 'armv7a-vfp-neon'
+    opkg_feed = 'http://www.tortall.net/~robotpy/feeds/2016/'
```
Comment (virtuald) [inline installer/installer.py]: This doesn't seem to be used anymore, since you're hardcoding it below?

**Comment by computer-whisperer** [comment]
What urllib import? I didn't add any.

File: installer/installer.py (hunk 1 above)
Comment (computer-whisperer) [inline installer/installer.py]: Right, I'll fix this soon.

File: installer/installer.py [hunk 2]
```diff
@@ -30,7 +31,7 @@
 
 from collections import OrderedDict
 from distutils.version import LooseVersion
-from urllib.request import urlopen, urlretrieve
```
Comment (virtuald) [inline installer/installer.py]: Oh, ha. I bet I added this. Ignore me.

**Comment by virtuald** [comment]
I like it, seems to work. I'm going to add a blacklist for system packages like libc6 though, as they aren't necessary and just eat up more disk space.

In the future I wonder if we should be smarter about the files that we copy via opkg, and if we should clean the cache on the roborio side. They can take up quite a bit of space.

**Pull request merged**